src/pages/ResetPassword.jsx

```diff
@@ -0,0 +1,96 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable import/no-unresolved */
+import { css } from '@emotion/react';
+import { PacmanLoader } from 'react-spinners';
+import React, { useState, useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+import { ToastContainer } from 'react-toastify';
+import Input from '../components/Auth/PasswordInput';
+import Button from '../components/Button/PasswordButton';
+import LoginIllustation from '../assets/login-illustration.svg';
+import { processReset } from '../redux/reducers/Auth/ResetPasswordSlice';
+import LoadingSpinner from '../components/LoadingSpinner';
+
+const override = css`
+	display: block;
+	margin: 0 auto;
+	text-align: center;
+`;
+
+const ResetPassword = () => {
+	const [password, setPassword] = useState('');
+	const [confirmPassword, setConfirmPassword] = useState('');
+	const token = document.cookie.split('=')[1];
+	const dispatch = useDispatch();
+	const { loading, error, success } = useSelector((state) => ({
+		loading: state.resetPassword?.loading,
+		error: state.resetPassword?.error,
+		succes: state.resetPassword?.succes,
+	}));
+
+	const navigate = useNavigate();
+	const handleProcessReset = (event) => {
+		event.preventDefault();
+		dispatch(processReset({ token, password, confirmPassword }));
+	};
+
+	useEffect(() => {
+		let timer;
+		if (success) {
+			timer = setTimeout(() => navigate('/login'), 5000); // Redirect to login page after 5 seconds
+		} else if (error) {
+			timer = setTimeout(() => window.location.reload(), 5000); // Reload the page after 5 seconds if there is an error
+		}
+
+		return () => clearTimeout(timer);
+	}, [success, error, navigate]);
+
+	return (
+		<div className="w-screen h-screen flex flex-row space-x-5 items-center">
+			<div className="hidden w-646 md:flex items-center justify-center">
+				<img
+					src={LoginIllustation}
+					alt=""
+					className="w-592 left-35 h-592 top-216"
+				/>
+			</div>
+			<div className="w-5/6 flex flex-col h-full bg-brightGray   h-1024 p-9 left-346 py-40 ">
+				<h3 className="text-left font-extrabold text-2xl font-bold text-yellow py-4">
+					Reset Password
+				</h3>
+				<form onSubmit={handleProcessReset}>
+					<Input
+						placeholder="password"
+						value={password}
+						type="password"
+						name="password"
+						onChange={(event) => setPassword(event.target.value)}
+						autoComplete="new-password"
+					/>
+					<Input
+						placeholder="confirm password"
+						value={confirmPassword}
+						type="password"
+						name="confirmPassword"
+						onChange={(event) => setConfirmPassword(event.target.value)}
+						autoComplete="new-password"
+					/>
+					{/* <div className="items-center">
+						{loading && (
+							<PacmanLoader color="#36D7B7" css={override} size={12} />
+						)}
+					</div> */}
```

Comment:
- use `className` prop to set input styles keeping in mind input width and height should be the same.
- check the values of left and right side division of the screen also.